vaccine_feed_ingest/runners/ak/clinic_list/fetch.yml

```diff
@@ -0,0 +1,6 @@
+---
+state: ak
+arcgis:
+- id: 7b65a9e852524227be91ad45d613da33
+  layer_names:
+  - COVID19_Vaccine_Site_Survey_PROD
```

Comment:
I wonder if this is a duplicate of [`ak/arcgis`](https://github.com/CAVaccineInventory/vaccine-feed-ingest/blob/main/vaccine_feed_ingest/runners/ak/arcgis/fetch.yml)? The layer name is the same.

Reply:
I've seen that layer name elsewhere, too, which makes me think there's a shared contractor doing this....